tests/rptest/tests/partition_force_reconfiguration_test.py

```diff
@@ -807,15 +812,15 @@ def test_recovery_local_data_missing(self):
         node_to_stop = self.redpanda.get_node_by_id(node_to_stop_id)
         self.logger.debug(f"Stopping node: {node_to_stop_id}")
         self.redpanda.stop_node(node_to_stop)
-
+        msg_size = 512
+        total_bytes = 75 * 1024 * 1024  # 75 MiB
+        msg_cnt = total_bytes // msg_size
         # produce more data while node 0 is stopped
-        producer = KgoVerifierProducer(self.test_context, self.redpanda,
-                                       topic.name, 512, 5000)
-        producer.start(clean=False)
-        producer.wait()
-        producer.clean()
-        producer.free()
-        self.wait_for_final_manifest_uploads(topic.name)
+        KgoVerifierProducer.oneshot(self.test_context, self.redpanda,
+                                    topic.name, msg_size, msg_cnt)
+
+        if wait_for_final_manifest_uploads:
+            self.wait_for_final_manifest_uploads(topic.name)
 
         status_2nd_step = self.get_topic_partition_status(topic.name, 1)
 
```

Comment:
nit: the rest of the test can be simplified a bit given we have only one partition now